<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lineTransWidget</class>
 <widget class="QWidget" name="lineTransWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>像素点线性和非线性变换</string>
  </property>
  <widget class="QPushButton" name="openImgBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开图像</string>
   </property>
  </widget>
  <widget class="QLabel" name="src_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tran_label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tran_label2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tran_label3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tran_label4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="formula_no1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>    y = 2x-55</string>
   </property>
  </widget>
  <widget class="QLabel" name="formula_no2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>    y = 0.5x-55</string>
   </property>
  </widget>
  <widget class="QLabel" name="formula_no3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>700</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>    y = x+55</string>
   </property>
  </widget>
  <widget class="QLabel" name="formula_no4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>700</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>    y = -x+255</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lineTransBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>线性变换</string>
   </property>
  </widget>
  <widget class="QLabel" name="tran_label5">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>240</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tran_label6">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>470</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="formula_no5">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>200</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>伽马变换</string>
   </property>
  </widget>
  <widget class="QLabel" name="formula_no6">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>700</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>对数变换</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
